SBR.md

```diff
@@ -2198,4 +2207,4 @@ The DNS TXT record MUST be placed on the "`_validation-contactemail`" subdomain
 
 ### A.3.2 DNS TXT Record Phone Contact
 
-The DNS TXT record MUST be placed on the "`_validation-contactphone`" subdomain of the domain being validated. The entire RDATA value of this TXT record MUST be a valid Global Number as defined in RFC 3966, Section 5.1.4, or it cannot be used.
\ No newline at end of file
+The DNS TXT record MUST be placed on the "`_validation-contactphone`" subdomain of the domain being validated. The entire RDATA value of this TXT record MUST be a valid Global Number as defined in RFC 3966, Section 5.1.4, or it cannot be used.
```

Comment:
Is this change intentional?

Reply:
Ah, yes, adding a newline would be good. The GitHub PR review UI doesn't render adding whitespace, so I was unsure what changed.